Souvenir Update bug fix and stadium complete

Wonton-Jon opened the pull request:
Update stadium complete
Bug fix where update souvenir button would delete last 5 souvenirs from datatable (they would still be in database just not on table)

Wonton-Jon committed:
Select data

Wonton-Jon committed:
Souvenir Update bug fix and stadium complete

Update stadium complete
Bug fix where update souvenir button would delete last 5 souvenirs from datatable (they would still be in database just not on table)

Wonton-Jon committed:
Souvenir Update bug fix and stadium complete

Update stadium complete
Bug fix where update souvenir button would delete last 5 souvenirs from datatable (they would still be in database just not on table)

Wonton-Jon committed:
Merge remote-tracking branch 'origin/feat/admin-update-stadium' into feat/admin-update-stadium

snowsk8r commented on app/Admin.cpp:
```diff
@@ -242,11 +234,6 @@ void Admin::on_Add_Souvenir_PushButton_clicked()
 	double    price        = ui -> Price_Double_SpinBox   -> text().toDouble();
 	int       teamID       = ui -> Team_Name_ComboBox -> currentIndex() + 1;
```
use combo box getItemData to get the teamID

snowsk8r commented on app/Admin.cpp:
```diff
@@ -29,9 +27,6 @@ Admin::Admin(QWidget *parent) :
 	PopulateComboBoxes("SELECT roofType FROM teamInfo"   , ui -> Roof_Type_ComboBox);
 	PopulateComboBoxes("SELECT teamName FROM teamInfo"   , ui -> Team_Name_ComboBox);
```
select teamID, teamName from teamInfo

snowsk8r commented on app/Admin.cpp:
```diff
@@ -29,9 +27,6 @@ Admin::Admin(QWidget *parent) :
-
 }
 
 //Populates all combo boxes with relaed information
```
line 51: use query.value(0) get ID, query.value(1) to get name
line 58: use combobox.setItemData() to add the ID to the combobox

snowsk8r commented on app/Admin.cpp:
```diff
@@ -154,7 +150,6 @@ void Admin::on_Read_In_From_File_Button_clicked()
-		QSqlQueryModel *model = nullptr;
 		QSqlQuery       query;
 		int             teamID = 33;
 		int             newSouvenirID = (teamID - 1) * 5;
```
let the database assign the ID instead of trying to calculate it

snowsk8r commented on app/Admin.cpp:
```diff
@@ -280,14 +267,19 @@ void Admin::on_Update_Souvenir_PushButton_clicked()
 	double    price        = ui -> Price_Double_SpinBox   -> text().toDouble();
 	int       teamID       = ui -> Team_Name_ComboBox     -> currentIndex() + 1;
```
use combo box getItemData to get the teamID

snowsk8r reviewed (dismissed):
Please review suggestions for changes.

Wonton-Jon committed:
Update added team fixed

Wonton-Jon committed:
Update error fixed

Wonton-Jon committed:
Bug fix where added items would remove from table

Wonton-Jon committed:
Purchases table

Purchases made will display:
-purchaseID
-team Name
-souvenirName
-quantity purchased

snowsk8r commented:
Looks fantastic!!  I tried to break it: deleted Arizona souvenirs before adding the team, all good.  With those deleted souvenirs, ran Green Bay to Arizona - only showed souvenirs to origin stadium (not AZ's).

snowsk8r reviewed (approved):
Awesome!!
